packages/grid/_modules_/grid/locales/ukUA.ts

```diff
@@ -56,65 +68,73 @@ const ukUAGrid: Partial<GridLocaleText> = {
   filterOperatorStartsWith: 'починається з',
   filterOperatorEndsWith: 'закінчується на',
   filterOperatorIs: 'дорівнює',
-  filterOperatorNot: 'не',
+  filterOperatorNot: 'не дорівнює',
   filterOperatorAfter: 'більше ніж',
   filterOperatorOnOrAfter: 'більше або дорівнює',
   filterOperatorBefore: 'менше ніж',
   filterOperatorOnOrBefore: 'менше або дорівнює',
-  // filterOperatorIsEmpty: 'is empty',
-  // filterOperatorIsNotEmpty: 'is not empty',
+  filterOperatorIsEmpty: 'порожній',
+  filterOperatorIsNotEmpty: 'не пустий',
 
   // Filter values text
   filterValueAny: 'будь-який',
   filterValueTrue: 'правда',
-  filterValueFalse: 'помилковий',
+  filterValueFalse: 'брехня',
 
   // Column menu text
   columnMenuLabel: 'Меню',
   columnMenuShowColumns: 'Показати стовпці',
   columnMenuFilter: 'Фільтр',
-  columnMenuHideColumn: 'Сховати',
+  columnMenuHideColumn: 'Приховати',
   columnMenuUnsort: 'Скасувати сортування',
   columnMenuSortAsc: 'Сортувати за зростанням',
   columnMenuSortDesc: 'Сортувати за спаданням',
 
   // Column header text
-  columnHeaderFiltersTooltipActive: (count) =>
-    count !== 1 ? `${count} активні фільтри` : `${count} активний фільтр`,
+  columnHeaderFiltersTooltipActive: getPluralFormForFilter,
   columnHeaderFiltersLabel: 'Показати фільтри',
   columnHeaderSortIconLabel: 'Сортувати',
 
   // Rows selected footer text
-  footerRowSelected: (count) =>
-    count !== 1
-      ? `${count.toLocaleString()} вибрані рядки`
-      : `${count.toLocaleString()} вибраний рядок`,
+  footerRowSelected: (count) => {
```

Comment:
Couldn't we reuse `getPluralFormForFilter` here? The different values for `pluralForm` could be passed as arguments.

Reply:
@m4theushw Sorry, I would like to clarify. Are you suggesting adding 3 more parameters or an object with 3 properties?

Reply:
> Are you suggesting adding 3 more parameters

Yes, just to reuse the same logic for the plural.

Reply:
I agree with Matheus.
There's general pluralisation rule in Ukrainian, so it would be great to have a generic function for that

Reply:
Please look at the changed function. Does it look good?